### PR TITLE
Created weblogic.xml

### PR DESCRIPTION
**Pull request by ajshastri**
This is to enable jenkins to work with weblogic.

# Description

Details: Added weblogic.xml with context jenkins for application weblogic support

### Changelog entries

Proposed changelog entries:

* Entry 1: Added weblogic.xml with context jenkins for application weblogic support

### CODE REVIEW

File: war/src/main/webapp/WEB-INF/weblogic.xml [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+<wls:weblogic-web-app
+xmlns:wls="http://xmlns.oracle.com/weblogic/weblogic-web-app"
+xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+xsi:schemaLocation="http://java.sun.com/xml/ns/javaee
+http://java.sun.com/xml/ns/javaee/ejb-jar_3_0.xsd
+http://xmlns.oracle.com/weblogic/weblogic-web-app
+http://xmlns.oracle.com/weblogic/weblogic-web-app/1.7/weblogic-web-app.xsd">
+
+     <wls:weblogic-version>12.1.3</wls:weblogic-version>
```
Comment (oleg-nenashev) [inline war/src/main/webapp/WEB-INF/weblogic.xml]: Should this version be hardcoded?

Comment (ajshastri) [inline war/src/main/webapp/WEB-INF/weblogic.xml]: Unfortunately yes. There is no harm from what I have experienced with it being a different version.

Comment (goughy000) [inline war/src/main/webapp/WEB-INF/weblogic.xml]: According to [this page](https://docs.oracle.com/cd/E24329_01/web.1211/e21049/weblogic_xml.htm#WBAPP574)

> The weblogic-version element indicates the version of WebLogic Server on which this Web application (as defined in the root element weblogic-web-app) is intended to be deployed. This element is informational only and is not used by WebLogic Server.

Is this entire element actually required?

**Review by oleg-nenashev — CHANGES_REQUESTED** [review]
Needs fix according to the comment from 
@goughy000

**Comment by oleg-nenashev** [comment]
@ajshastri any plans to address comments? If no, I propose to close the PR

**Pull request closed without merging**